app/views/alaveteli_pro/plans/index.html.erb

```diff
@@ -6,8 +6,13 @@
     <div class="pricing__tier__heading">
       <h2><%= _('Professional') %></h2>
       <p class="pricing__tier__subhead"><%= _('For journalists, academics and power users') %></p>
-      <p class="price-label"><%= _('<span class="price-label__amount">{{monthly_price}}</span> per user, per month</p>',
-                                   monthly_price: format_currency(@plan.amount_with_tax, true)) %>
+      <p class="price-label">
+        <%= _('<span class="price-label__amount">{{monthly_price}}</span> ' \
+              'per user, per month',
+              monthly_price: format_currency(
+                               @plan.amount_with_tax,
+                               no_cents_if_whole: true)) %>
+      </p>
```

Comment:
Or - given the assumptions in the layout - can we split that in e.g. `£10` (no translation required as `format_currrency` is doing the work here) and `_('per user, per month')`?

Reply:
I think keeping the full context `"{{monthly_price}} per user, per month"` is good in case it makes more sense to invert the price/text in some languages (e.g. `"per user, per month {{monthly_price}}"`).